bdio-protobuf/build.gradle

```diff
@@ -0,0 +1,35 @@
+description = 'Black Duck I/O protobuf format definitions'
+
+apply plugin: 'com.google.protobuf'
+
+dependencies {
+   compile 'com.google.protobuf:protobuf-java:3.20.3'
```

Comment:
Can the protobuf-java version be moved to the dependencies..properties file?

Reply:
sure.